partner_credit_limit/model/sale.py

```diff
@@ -54,21 +55,23 @@ def check_limit(self):
                 return True
             else:
                 msg = _('<div><p>The Sale order pass to state of '
-                        'Exception Credit.\n '
-                        '<br>The partner %s:') % (partner.name)
+                        '<b>Exception Credit</b>.\n '
+                        '<br>The partner <b>%s</b>:') % (partner.name)
                 if partner.credit_overloaded:
                     msg += _('\n<br><br>Have exceeded the credit limit.'
-                             '\n<br>The credit available is $%s'
+                             '\n<br>The credit available is <b>$%s</b>'
                              '\n<br>And the credit is being'
-                             ' requested is $%s') % \
-                        (str((partner.credit_limit - partner.credit)),
+                             ' requested is <b>$%s</b>') % \
+                        (str(round((partner.credit_limit - partner.credit))),
                          str(so.amount_total))
                 if partner.overdue_credit:
                     max_date = max(self.partner_overdue_payments_ids.mapped(
                         'date_maturity'))
+                    max_date = datetime.strptime(
+                        max_date, '%Y-%m-%d').strftime('%d-%m-%Y')
                     msg += _('\n<br><br>It has the overdue payment period.'
-                             '\n<br>The expiration date was %s, '
-                             '<br>the amount payable is: $%s') % \
-                        (max_date, str(partner.credit))
+                             '\n<br>The expiration date was <b>%s</b>, '
+                             '<br>the amount payable is: <b>$%s</b>') % \
+                        (max_date, str(self.partner_overdue_amount))
```

Comment:
Don't ask me, try it yourself: 
<img width="154" alt="screen shot 2016-04-20 at 1 30 34 pm" src="https://cloud.githubusercontent.com/assets/6644187/14685895/1ca95356-06fc-11e6-9411-b864b620b0ca.png">